### DISCUSSION TITLE
How to apply RTL

### DISCUSSION THREAD

**Discussion by RitikPatle** [daohoangson/flutter_widget_from_html#883] [Q&A]
How to apply text Direction as rtl or ltr in this widget.
that tag is not working I suppose

**Comment by ghost** [comment]
use [Directionality](https://api.flutter.dev/flutter/widgets/Directionality-class.html) widget.
if you want to dynamically determine text direction based on content use [intl's Bidi class `detectRtlDirectionality`](https://api.flutter.dev/flutter/intl/Bidi/detectRtlDirectionality.html) method. example:
```dart
	final textDirection =
		intl.Bidi.detectRtlDirectionality(htmlString, isHtml: true)
            ? TextDirection.rtl
            : TextDirection.ltr;

	return Directionality(
		textDirection: textDirection,
			child: HtmlWidget(htmlString),
    );
```

**Comment by daohoangson** [comment]
`Directionality` widget or `dir` attribute should work.
